jetty-util/src/main/java/org/eclipse/jetty/util/resource/URLResource.java

```diff
@@ -290,7 +290,9 @@ public Resource addPath(String path)
         throws IOException, MalformedURLException
     {
         if (path == null)
-            return null;
+        {
+            throw new MalformedURLException("null path");
```

Comment:
`IllegalArgException`

Reply:
How about replacing that entire thing with `Objects.requireNotNull(path)` ?

Reply:
sure